newrelic.yml

```diff
@@ -102,7 +102,7 @@ common: &default_settings
   # Specify a list of constants that should prevent the agent from starting
   # automatically. Separate individual constants with a comma ,. For example,
   # "Rails::Console,UninstrumentedBackgroundJob".
-  # autostart.denylisted_constants: Rails::Console
+  # autostart.denylisted_constants: Rails::Command::ConsoleCommand,Rails::Command::CredentialsCommand,Rails::Command::Db::System::ChangeCommand,Rails::Command::DbConsoleCommand,Rails::Command::DestroyCommand,Rails::Command::DevCommand,Rails::Command::EncryptedCommand,Rails::Command::GenerateCommand,Rails::Command::InitializersCommand,Rails::Command::NotesCommand,Rails::Command::RoutesCommand,Rails::Command::SecretsCommand,Rails::Console,Rails::DBConsole
```

Comment:
Since we have automation to update newrelic.yml, we will lose these changes unless updated in `default_source`!

Reply:
Thanks @hannahramadan. The changes made to `newrelic.yml` in this PR were from running the rake task to autogenerate a new file. I likely should have simply withheld the changes from the PR, but it's also not going to hurt anything to have it. I kinda favor `newrelic.yml` be generated on a per PR basis in the future. The `default_source.rb` file has all of the content I need, and I'm okay with the pending `newrelic.yml` regeneration driven by the next release.

Reply:
Ooh totally! Agree that running on a PR basis sounds nicer. Having that workflow run on only pushes to `main` leaves room for surprises we might not want. Making a ticket for this.